addon/styles/components/cta.scss

```diff
@@ -0,0 +1,28 @@
+$btn-primary: #1a1a1a;
+$btn-primary--hover: #5d5d5d;
+$btn-secondary: #a3a3a3;
```

Comment:
Not sure if these are the best variable names, but I'm trying to avoid having names like `gray`, `light-gray`, `light-gray-2`

Reply:
how about something like:
```
$light-gray: #e8e8e8;
...
$btn-primary-hover: $light-gray;
```